lib/osf-components/addon/components/registries/overview-form-renderer/component.ts

```diff
@@ -0,0 +1,26 @@
+import { tagName } from '@ember-decorators/component';
+import Component from '@ember/component';
+import { task } from 'ember-concurrency';
+import { layout } from 'ember-osf-web/decorators/component';
+import Registration from 'ember-osf-web/models/registration';
+import { getSchemaBlockGroups, SchemaBlockGroup } from 'ember-osf-web/packages/registration-schema';
+import template from './template';
+
+@tagName('')
+@layout(template)
+export default class RegistrationFormViewSchemaBlocks extends Component.extend({
+    fetchSchemaBlocks: task(function *(this: RegistrationFormViewSchemaBlocks) {
+        if (this.registration) {
+            const registrationSchema = yield this.registration.registrationSchema;
+            const schemaBlocks = yield registrationSchema.loadAll('schemaBlocks');
```

Comment:
hm, it'd be nice if we could use the schema blocks that might already be in the store, instead of *always* waiting on a request. maybe
```ts
const schemaBlocksRef = registrationSchema.hasMany('schemaBlocks');
const schemaBlocks = schemaBlocksRef.value() || yield registrationSchema.loadAll('schemaBlocks');
```
?

Reply:
This doesn't seem to work for non-async relationships and `schemaBlocks` is a non-async relationship of `registration-schema`. `schemaBlocksRef.value()` always return an object.